### ISSUE TITLE
Frontend: db backup

### ISSUE THREAD

**Issue by benallfree** [benallfree/pockethost#95]
@brewhousedigital Following #85, we will need a substantial cosmetic pass if you are willing :)

- [ ] Warn on restore: instance must be shut down
- [ ] Warn on restore: if instance will change platforms

**Comment by brewhousedigital** [comment]
Sure thing! Is it ready to be looked at now? 

When and where do these events happen?

**Comment by benallfree** [comment]
Yeah #85 is still wip, I will circle back to this issue when it's ready to go. For now it's on hold. 

I might drop db restores from it, not sure yet!

**Comment by benallfree** [comment]
@brewhousedigital scoping this down to just backup. You are clear to proceed, 0.5.0 has been deployed.

**Comment by benallfree** [comment]
@brewhousedigital after #106

**Comment by benallfree** [comment]
moved to roadmap
